src/lib/Dungeon.js

```diff
@@ -264,15 +296,38 @@ class AutomationDungeon
             if (this.__internal__isFirstMove)
             {
                 this.__internal__ensureTheBoardIsInAnAcceptableState();
+                if (DungeonRunner.map.flash)
+                {
+                    this.__internal__markNearbyTiles();
+                }
```

Comment:
wouldn't this be already set bu the `this.__internal__ensureTheBoardIsInAnAcceptableState();` call ?